src/views/poi_bloc/Accessibility.jsx

```diff
@@ -3,13 +3,24 @@ import React from 'react';
 
 const AccessibilityBlock = ({
   block: accessibilityList,
-  _poi,
-  options: accessibilities,
   asString,
 }) => {
-  const availableAccessibilities = accessibilities.map(a11y =>
-    _(a11y.labels[accessibilityList[a11y.key]])
-  );
+  const labels = {
+    'wheelchair': {
+      'yes': _('Wheelchair accessible'),
+      'partial': _('Partially wheelchair accessible'),
+      'no': _('Not wheelchair accessible'),
+    },
+    'toilets_wheelchair': {
+      'yes': _('Wheelchair accessible toilets'),
+      'partial': _('Partial wheelchair accessible toilets'),
+      'no': _('No wheelchair accessible toilets'),
+    },
+  };
+  const availableAccessibilities = [];
+  for (const label in labels) {
+    availableAccessibilities.push(_(labels[label][accessibilityList[label]]));
```

Comment:
Now that everything has been regrouped in the component, do we really need this mess that tries to be generic? We can probably do that in a more readable way.
Also, I think there are now two calls to the `_` function.

Reply:
Good catch for the double `_` calls!

About the generic part, it's still needed. We make the display depending on the argument we receive after all.

Reply:
Ok, I guess we can improve that later if it's needed, as long as it's self contained this isn't a big deal. But I still find this pretty ugly :) (not your fault though)